qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -160,10 +161,14 @@ def create_reqd_users(self):
                                              '123456-fsgqa'], omit_sudo=False,
                                        check_status=False)
 
-    def write_local_config(self):
+    def write_local_config(self, options=None):
         from os.path import join
         from textwrap import dedent
 
+        _options = ''
+        if options:
+            _options = options + ','
+
```

Comment:
nit:
This would be more succinct and readable too.
```suggestion
        _options = '' if options else options + ','

```

Reply:
Looks better and I will fix it.